xml/System.IO/TextWriter.xml

```diff
@@ -1570,9 +1580,17 @@ This member is an explicit interface member implementation. It can be used only
         <Parameter Name="value" Type="System.Text.StringBuilder" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="value">The string, as a string builder, to write to the text stream.</param>
+        <summary>Writes a string builder to the text stream.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Equivalent to `Write(stringBuilder.ToString())`, but uses the `StringBuilder.GetChunks()` method to avoid creating the intermediate string.
```

Comment:
```suggestion
This method is equivalent to `Write(stringBuilder.ToString())`, but it uses the <xref:System.Text.StringBuilder.GetChunks%2A?displayProperty=nameWithType>  method to avoid creating the intermediate string.
```